collated/test/A0139552B.md

```diff
@@ -0,0 +1,169 @@
+# A0139552B
+###### \java\guitests\EditCommandTest.java
+``` java
+    @Test
+    public void edit() {    	
+    	
+        TestTask[] currentList = td.getTypicalTasks();
+        assertClearCommandSuccess();
+
+        TestTask aliceTask = new TestTask(td.alice);
+        assertAddSuccess(aliceTask);
+        
+        //test to check each parameter
+        commandBox.runCommand("edit 1 Call Alice from 2pm to 3pm repeat every day -high");
+        aliceTask.setName(new Name("Call Alice"));
+        aliceTask.setStartDate(DateTime.convertStringToDate("2pm"));
+        aliceTask.setEndDate(DateTime.convertStringToDate("3pm"));
+        try {
```

Comment:
no need to `try ... catch` exceptions in test classes; you can simply `throw` the exception